pkg/lib/cockpit.js

```diff
@@ -1049,7 +1049,7 @@ function factory() {
         self.encoding = "utf-8";
 
         self.decode = function decode(data, options) {
-            const stream = options && options.stream;
+            const stream = options?.stream;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18448-20230311-055523-7e3e6fe9-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)